### ISSUE TITLE
ManiDp is not work

### ISSUE THREAD

**Issue by ZhangLei999** [huawei-noah/GAN-pruning#13]
After run the command:
python ManiDP/main.py --ngpu=1 --arch=dyresnet56 --dataset=cifar10 --target_remain_rate=0.6 --lambda_lasso 5e-3 --lambda_graph 1.0 --pretrain_path='ManiDP/pretrain_path/' --data_path='...'
the loss is nan.
I think this error is caused by the lasso loss. Can you give some advice. Thanks

**Comment by Parsifal133** [comment]
他这个pretrain自己训练吗？

**Comment by zhichengye7** [comment]
> After run the command: python ManiDP/main.py --ngpu=1 --arch=dyresnet56 --dataset=cifar10 --target_remain_rate=0.6 --lambda_lasso 5e-3 --lambda_graph 1.0 --pretrain_path='ManiDP/pretrain_path/' --data_path='...' the loss is nan. I think this error is caused by the lasso loss. Can you give some advice. Thanks

I have the same problem. Have you solved it now？

**Comment by yehuitang** [comment]
Thanks for your attention to our work.  Have you use the pre-trained models?

**Comment by ZhangLei999** [comment]
Yes, the pre-trained model is trained by myself. If the architecture is dyresnet20, the loss is normal. Others does not work

**Comment by yehuitang** [comment]
The loss is normal when I run the code.  I guess that the pre-trained models may has impact. Maybe you can try to reduce the learning rate and lambda_lasso or clip gradients to make the training stable?

**Comment by whz945565** [comment]
> Yes, the pre-trained model is trained by myself. If the architecture is dyresnet20, the loss is normal. Others does not work
你好，请问自己训练怎么训练的，是把那两个参数设置为0吗

**Comment by NoLookDefense** [comment]
> Yes, the pre-trained model is trained by myself. If the architecture is dyresnet20, the loss is normal. Others does not work

Hello. Could you provide your pre-trained model? The author didn't provide it.
